### PR TITLE
Enable bloom in zone properties

### PR DESCRIPTION
**Pull request by SamGondelman**
https://highfidelity.manuscript.com/f/cases/15034/Add-bloom-control-to-the-Zone-UI-in-Create

Test plan:
- Go to your sandbox and go to an area with some bright areas.
- Create a zone.  In the zone properties, set Bloom to ON.
- In Running Scripts, search for and run debugBloom.js.  Select "All".
- You should see something like this:
bloom1
- As you increase and decrease the Bloom Intensity, the bottom right image should get brighter and darker, respectively.
- As you increase and decrease the Bloom Threshold, all four images should get darker and brighter, respectively.
- As you increase and decrease the Bloom Size, all four images should get more and less blurred, respectively.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11932/HighFidelity-Beta-PR13778-2959a40.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11932/HighFidelity-Beta-PR13778-2959a40.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11932/HighFidelity-Beta-Interface-PR13778-2959a40.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11932/HighFidelity-Beta-PR13778-2959a40.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11932/HighFidelity-Beta-Interface-PR13778-2959a40.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11950/HighFidelity-Beta-PR13778-e4aa3e6.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11950/HighFidelity-Beta-PR13778-e4aa3e6.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11950/HighFidelity-Beta-Interface-PR13778-e4aa3e6.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11957/HighFidelity-Beta-PR13778-a0c3c09.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11957/HighFidelity-Beta-PR13778-a0c3c09.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11957/HighFidelity-Beta-Interface-PR13778-a0c3c09.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11957/HighFidelity-Beta-PR13778-a0c3c09.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11957/HighFidelity-Beta-Interface-PR13778-a0c3c09.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11967/HighFidelity-Beta-PR13778-bba806f.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11967/HighFidelity-Beta-PR13778-bba806f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11967/HighFidelity-Beta-Interface-PR13778-bba806f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11967/HighFidelity-Beta-PR13778-bba806f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11967/HighFidelity-Beta-Interface-PR13778-bba806f.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Tested on Mac and Windows. Bloom appears well.

**Comment by Menithal** [comment]
How does the bloom behave with haze?

**Comment by SamGondelman** [comment]
@Menithal bloom is a post-processing effect, like AA, which is applied after haze.  It should work as expected (i.e. bright colors blocked by haze will not have bloom).

**Comment by conklin94122** [comment]
73.0 PRs can not be QA Approved until they have a branch to be merged into. Testers will retest once we cut the 72.0 branch allowing 73.0 PRs to be merged into master. fyi @qlimnavarro

File: libraries/entities/src/ZoneEntityItem.cpp [hunk 1]
```diff
@@ -47,33 +47,27 @@ ZoneEntityItem::ZoneEntityItem(const EntityItemID& entityItemID) : EntityItem(en
 EntityItemProperties ZoneEntityItem::getProperties(EntityPropertyFlags desiredProperties) const {
     EntityItemProperties properties = EntityItem::getProperties(desiredProperties); // get the properties from our base class
 
-    // Contains a QString property, must be synchronized
+    // Contain QString properties, must be synchronized
```
Comment (NissimHadar) [inline libraries/entities/src/ZoneEntityItem.cpp]: Just curious - why did this change?

Comment (SamGondelman) [reply libraries/entities/src/ZoneEntityItem.cpp]: I grouped the calls that need locks so that we're only doing one.

**Comment by qaAdamSmith82** [comment]
Currently Testing

**Comment by qaAdamSmith82** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12390/HighFidelity-Beta-PR13778-bba806f.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12390/HighFidelity-Beta-PR13778-bba806f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12390/HighFidelity-Beta-Interface-PR13778-bba806f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12390/HighFidelity-Beta-PR13778-bba806f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12390/HighFidelity-Beta-Interface-PR13778-bba806f.exe

**Comment by qaAdamSmith82** [comment]
@conklin94122 PASS

**Pull request merged**